client/components/redux-forms/redux-form-radio/index.jsx

```diff
@@ -17,6 +17,7 @@ class ReduxFormRadio extends Component {
 
 	renderRadio = defaultValue => ( { input: { name, onChange, value } } ) => (
 		<FormRadio
+			{ ...this.props }
```

Comment:
Should we `omit` props that we need here but not from within `FormRadio`? E.g. this will pass `name` (which is explicitly passed two lines below) and `value` (which is overridden on L24 by `defaultValue`), which might cause some confusion. (Same for other `ReduxForm` components.)